src/main/external-resources/renderers/DefaultRenderer.conf

```diff
@@ -621,8 +621,10 @@ AutoExifRotate =
 #    mkv      Matroska Video                                video/x-matroska
 #    mlp      Meridian Lossless Packing                     audio/vnd.dolby.mlp
 #    mov      QuickTime File Format                         video/quicktime
+#    mp2      MPEG Audio Layer II                           audio/mpeg
```

Comment:
@Sami32 I'm a bit scared to mess with video, there are so many "hacks" in UMS related to video that the result of changing anything is hard to predict. As this is only an edit to ```DefaultRenderer.conf``` it shouldn't be a big deal though, but it should be tested if these will actually work first.